lib/cpp/test/SecurityTest.cpp

```diff
@@ -108,8 +108,13 @@ struct SecurityFixture : public TestPortFixture
 
             pServerSocketFactory.reset(new TSSLSocketFactory(static_cast<apache::thrift::transport::SSLProtocol>(protocol)));
             pServerSocketFactory->ciphers("ALL:!ADH:!LOW:!EXP:!MD5:@STRENGTH");
+          #ifdef BOOST_WINDOWS_API
+            pServerSocketFactory->loadCertificate(certFile("server.crt").string().c_str());
+            pServerSocketFactory->loadPrivateKey(certFile("server.key").string().c_str());
+          #else
             pServerSocketFactory->loadCertificate(certFile("server.crt").native().c_str());
             pServerSocketFactory->loadPrivateKey(certFile("server.key").native().c_str());
+          #endif
             pServerSocketFactory->server(true);
```

Comment:
I don't get it. You are using "string" not "wstring" method.
"string" will be fine for Unix too, so I'd like to keep your version only, removing original "native".